src/test/rgw/rgw_multi/tests.py

```diff
@@ -252,7 +252,7 @@ def bucket_sync_status(target_zone, source_zone, bucket_name):
     if target_zone == source_zone:
         return None
 
-    cmd = ['bucket', 'sync', 'status'] + target_zone.zone_args()
+    cmd = ['bucket', 'sync', 'markers'] + target_zone.zone_args()
```

Comment:
can we add a separate test for the new sync status?

Reply:
this wasn't really a test for sync status, it's part of the implementation of bucket checkpoints